cpp/src/transform/jit/kernel.cu

```diff
@@ -33,29 +36,89 @@ namespace cudf {
 namespace transformation {
 namespace jit {
 
-/// @brief This class supports striding into columns of data as either scalars or actual
-/// columns at no runtime cost. Although it implies the kernel will be recompiled if scalar and
-/// column inputs are interchanged.
-template <typename T, int multiplier>
-struct strided {
-  T data;
+template <typename T, int32_t index>
+struct accessor {
+  using type                     = T;
+  static constexpr int32_t INDEX = index;
```

Comment:
```suggestion
template <typename T, int32_t Index>
struct accessor {
  using type                     = T;
  static constexpr int32_t index = Index;
```
Our template paramters are [using UpperCamelCase naming](https://github.com/rapidsai/cudf/blob/branch-25.04/cpp/doxygen/developer_guide/DEVELOPER_GUIDE.md#code-and-documentation-style-and-formatting)

Reply:
Thanks! is there a reason we don't have lint for this?

Reply:
Not that I'm aware of. It's probably just one of the nice-to-have features we haven't added yet.